cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js

```diff
@@ -39,7 +38,7 @@ describe('ui-data-import: A user can delete import logs with "Data import: Can d
           const filePath = numberOfLogsToUpload - 1 === index ? filePathToUpload : emptyFilePathToUpload;
           fileNameToUpload = `C358137autotestFile.${getRandomPostfix()}.mrc`;
           DataImport.uploadFile(filePath, fileNameToUpload);
-          JobProfiles.searchJobProfileForImport(jobProfileToRun);
+          JobProfiles.searchJobProfileForImport('Default - Create instance and SRS MARC Bib');
```

Comment:
Please delete all remained logs in the after section . It is nearly 25 of logs with the filename starting "C358137autotestFile."

Reply:
Done